Description: Remove test_css_return_if_off()
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/844921
Forwarded: no
Last-Update: 2016-12-13

--- python-django-compressor-2.1.orig/compressor/tests/test_parsers.py
+++ python-django-compressor-2.1/compressor/tests/test_parsers.py
@@ -96,15 +96,6 @@ class Html5LibParserTests(ParserTestCase
                                                split[1][3].attrib,
                                                split[1][3].text))
 
-    @override_settings(COMPRESS_ENABLED=False)
-    def test_css_return_if_off(self):
-        # Yes, they are semantically equal but attributes might be
-        # scrambled in unpredictable order. A more elaborate check
-        # would require parsing both arguments with a different parser
-        # and then evaluating the result, which no longer is
-        # a meaningful unit test.
-        self.assertEqual(len(self.css), len(self.css_node.output()))
-
     @override_settings(COMPRESS_PRECOMPILERS=(), COMPRESS_ENABLED=False)
     def test_js_return_if_off(self):
         # As above.
diff --git a/compressor/tests/test_base.py b/compressor/tests/test_base.py
index cb8f52f..a0aa684 100644
--- a/compressor/tests/test_base.py
+++ b/compressor/tests/test_base.py
@@ -171,10 +171,6 @@ class CompressorTestCase(SimpleTestCase):
             self.assertTrue(is_date.match(str(float(date))),
                 "mtimes is returning something that doesn't look like a date: %s" % date)
 
-    @override_settings(COMPRESS_ENABLED=False)
-    def test_css_return_if_off(self):
-        self.assertEqualCollapsed(self.css, self.css_node.output())
-
     def test_cachekey(self):
         is_cachekey = re.compile(r'\w{12}')
         self.assertTrue(is_cachekey.match(self.css_node.cachekey),
